docs/src/explanations/chisel-enum.md

```diff
@@ -0,0 +1,133 @@
+# ChiselEnum
+
+The ChiselEnum type is an easy way to get away from value encodings and reduce the chance of error when encoding muxes, opcodes, and functional unit operations. As opposed to the Chisel type Enums, ChiselEnums allow for names to be explicitly passed through Bundles like IO. 
+
+## Importing the ChiselEnum module
+
+```scala mdoc
```

Comment:
can you break these up or make some of these invisible/silent since `ChiselStage` is not relevant to to "Importing the Chisel Enum module" ?

Reply:
I'm not sure the general pattern in these docs if things that are only relevant for the examples are hidden or if the code examples are all supposed to work right out of the box

Reply:
In referencing https://raw.githubusercontent.com/chipsalliance/chisel3/master/docs/src/explanations/naming.md they also had ChiselStage shown

Reply:
yeah, gotcha. Then let's follow that pattern a bit more ... I'll suggest a change

Reply:
I also believe this helps people who want to copy what is on screen and get it to run locally on their machine.